index.html

```diff
@@ -11,22 +11,34 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-  <header class="Toolbar">
-    <div class="logo">
-      <a href="index.html" class="Mylogo">JIC Software</a>
-    </div>
-    <div class="menu-pc">
-      <a class="item-menu-pc">Portfolio</a>
-      <a class="item-menu-pc">About</a>
-      <a class="item-menu-pc-group">
-      <a class="item-menu-pc">Contact</a>
-        <img class="item-menu-pc-icon" src="images/mail_icon.png" alt="email icon">
-      </a>
+  <header>
+    <div class="Toolbar">
+      <div class="logo">
+        <a href="#" class="Mylogo">JIC Software</a>
+      </div>
+      <div class="menu-pc">
+        <a href="#portfolio" class="item-menu">Portfolio</a>
+        <a href="#about" class="item-menu">About</a>
+        <div href="#contact" class="item-menu-group">
+          <a href="#contact" class="item-menu">Contact</a>
+          <img href="#contact" class="item-menu-icon" src="images/mail_icon.png" alt="email icon">
+        </div>
+      </div>
+      <nav id="burguer-menu" class="burguer-menu">
+          <img class="burguer-menu-icon" src="images/burguer_menu_icon.png" alt="burguer menu icon">
+      </nav>
     </div>
-    <nav class="menu-mobile">
-        <img class="burguer-menu" src="images/burguer_menu_icon.png" alt="burguer menu icon">
-    </nav>
+
+    <div class="menu-bar-movil">
+      <nav class="slidemenu" id="slidemenu">
+        <a class="btn-close-slide-menu">X</a>
+        <a href="#portfolio" class="item-menu-mobile">Portfolio</a>
+        <a href="#about" class="item-menu-mobile">About</a>
+        <a href="#contact" class="item-menu-mobile">Contact</a>
+      </nav>
+		</div>  <!-- menu movil-->    
```

Comment:
- Please stick to the original design as much as possible, which includes:
   - [ ] Adjusting the positioning of the nav menu elements and the close button to match the design.
   - [ ] Changing the color of the close button to match the Figma template.
   - [ ] Changing the close button size to match the design.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/232603295-b1ef6e8c-d278-4853-88fa-1c29eba0feec.png) | ![image](https://user-images.githubusercontent.com/62290545/232596283-5b2c7c38-785c-402c-9ee2-b09ac8f509a0.png) |